### PR TITLE
auth-4.6.0-beta1: secpoll&docs

### PR DESCRIPTION
**Pull request by Habbie**
~Unfinished, alpha2 might grow a few more PRs.~ Now finished, and renamed to beta.1

### CODE REVIEW

File: docs/changelog/4.6.rst [hunk 1]
```diff
@@ -1,6 +1,89 @@
...
+  This release contains a ton of improvements and bug fixes compared to 4.5, and a couple of user visible changes.
+
+  New features:
+
+  * support for incoming PROXY headers
+  * support for EDNS Cookies
+
+  Please make sure to read the :doc:`upgrade notes <../upgrading>` before upgrading.
+
+  Compared to 4.6.0-alpha2, the major user visible change is the new NSEC3PARAM settings (again, see the upgrade docs).
```
Comment (mnordhoff) [inline docs/changelog/4.6.rst:18]:
```suggestion
  Compared to 4.6.0-alpha1, the major user visible change is the new NSEC3PARAM settings (again, see the upgrade docs).
```

File: docs/secpoll.zone [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@       86400   IN  SOA pdns-public-ns1.powerdns.com. pieter\.lexis.powerdns.com. 2021112901 10800 3600 604800 10800
+@       86400   IN  SOA pdns-public-ns1.powerdns.com. pieter\.lexis.powerdns.com. 202121001 10800 3600 604800 10800
```
Comment (mnordhoff) [inline docs/secpoll.zone:1]: Serial number is missing a digit. :D

File: docs/changelog/4.6.rst [hunk 3]
```diff
@@ -1,6 +1,89 @@
...
+  Besides that, see below for various fixes.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 10982
+
+    Condition to HAVE_SYSTEMD_WITH_RUNTIME_DIR_ENV is reversed
```
Comment (pieterlexis) [inline docs/changelog/4.6.rst:25]:
```suggestion
    Condition to HAVE_SYSTEMD_WITH_RUNTIME_DIR_ENV is reversed
    During build, the runtime directory in the service files for virtual-hosting are now correctly generated.
```

File: docs/secpoll.zone [hunk 4]
```diff
@@ -1,4 +1,4 @@
-@       86400   IN  SOA pdns-public-ns1.powerdns.com. pieter\.lexis.powerdns.com. 2021120304 10800 3600 604800 10800
+@       86400   IN  SOA pdns-public-ns1.powerdns.com. pieter\.lexis.powerdns.com. 2021120901 10800 3600 604800 10800
```
Comment (rgacogne) [inline docs/secpoll.zone:1]:
```suggestion
@       86400   IN  SOA pdns-public-ns1.powerdns.com. pieter\.lexis.powerdns.com. 2021121001 10800 3600 604800 10800
```

**Review by rgacogne — APPROVED** [review]

File: docs/secpoll.zone (hunk 4 above)
Comment (Habbie) [reply docs/secpoll.zone:1]: ah, thanks! I fixed the changelog instead :):

File: docs/changelog/4.6.rst [hunk 5]
```diff
@@ -1,6 +1,108 @@
...
+    :tags: Improvements
+    :pullreq: 11067
+
+    pdnsutil: skip SHA1 DSes except in verbose mode
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 11066
+
+    add zone to zonecache on flush API endpoint
```
Comment (jsoref) [inline docs/changelog/4.6.rst:37]:
```suggestion
    add zone to zone cache on flush API endpoint
```

**Pull request merged**